map-prototype/index.html

```diff
@@ -0,0 +1,217 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no"/>
+  <title>Map / Philly Mesh</title>
```

Comment:
There are ways, but I think choosing a good one depends more on whatever like idiom we want to follow. We already work with jekyll, yaml, liquid for our web stuff so maybe continue in that vein?

Based on our current web deployment I could see either being feasible(?):
1. using jekyll and _config.yml, and having to generate the site before deploy 
2. using grunt with [grunt-liquid](https://www.npmjs.com/package/grunt-liquid) to have the grunt config handle titles, etc...